x-pack/test/apm_api_integration/tests/alerts/anomaly_alert.spec.ts

```diff
@@ -77,26 +78,37 @@ export default function ApiTest({ getService }: FtrProviderContext) {
       }
 
       describe('with ml jobs', () => {
-        it('checks if alert is active', async () => {
-          const createdRule = await createApmRule({
+        let ruleId: string;
+        let createdRule: any;
```

Comment:
Can you please add return types to `createApmRule`. A minimum definition like `Promise<{ id: string }>` is fine.

```suggestion
        let createdRule: any;
```